pages/faq.md

```diff
@@ -0,0 +1,182 @@
+---
+slug: faq
+title: Frequently Asked Questions
+description: Find answers to some of the frequently asked questions here.
+type: text
+---
+<style>
+h2 {
+  font-size: 2rem;
+  text-transform: none;
+}
+h3,
+h4,
+h5,
+h6 {
+  font-size: 1.125rem;
+  font-weight: normal;
+  text-transform: uppercase;
+}
+</style>
+Find answers related to the AWX project in the following section.
+You can also find lots of information about AWX in the [Ansible community forum](https://forum.ansible.com/tag/awx).
+
+**Core**  
+[Q: What is the difference between Ansible Core and AWX?](#1)  
+**AWX**  
+[Q: What is The AWX Project?](#2)  
+[Q: Why is Red Hat doing this?](#3)  
+[Q: What’s the difference between AWX and automation controller?](#4)  
+[Q: Where can I find support for AWX?](#5)  
+[Q: Can I upgrade from one version of AWX to another?](#6)  
+[Q: Under which open source license is AWX available?](#7)  
+[Q: How can I get involved with the AWX Project?](#8)  
+[Q: Where do I report an AWX bug?](#9)  
+[Q: How often is AWX released?](#10)  
+[Q: What is the governance structure of the AWX project?](#11)  
+[Q: If my software runs with AWX, can I say that it is certified to run on AWX or on Red Hat Ansible Automation Platform?](#12)  
+[Q: I want to build my own forked version of AWX. Can I call it AWX? Can I call it Red Hat Ansible Automation Platform?](#13)  
+[Q: Will contributions to AWX require a “Contributor License Agreement”?](#14)  
+[Q: Does Red Hat recommend AWX for production environments?](#15)  
+[Q: Does Red Hat’s Open Source Assurance Program apply to AWX?](#16)  
+**Ansible Community**  
+[Q: How can I contribute back to the community?](#17)  
+<br><br>
```

Comment:
These line breaks are unclosed tags.

Reply:
The <br> tag is an empty tag, which means that it has no end tag.  This is per HTML standards: https://www.w3schools.com/html/html_paragraphs.asp.  If you have a different understanding let me know.

Reply:
Maybe it's just a tick that I got from somewhere - probably from my days of xhtml transforms - but I was referring to `<br />` 

I always use the closing forward slash like that but maybe it's just an old habit. I'm not an HTML guru or anything.